### PR TITLE
Toggle Button Documentation

### PR DESCRIPTION
**Pull request by VinayakChopra**
This pull request adds the use of the toggle button as checkboxes and radio buttons into the documentation page as well as documents the same in the readme.md file.

Issue: Add documentation about toggle button #983

### CODE REVIEW

File: documentation.html [hunk 1]
```diff
@@ -352,6 +357,55 @@ <h4 class="display-inlineBlock">Icon Positioned to The Right</h4>
             </div>
           </section>
         </section>
+        <!-- Toggle button class starts here -->
+        <section class="instruction-block" id="toggle">
+          <h2 id="base-icon-btn">Toggle Button</h2>
+          <div class="text">
+            The <code>toggle-btn</code> class along with <code>toggle-off-btn</code> / <code>toggle-on-btn</code> state class enables you to have a toggle button.Apart from its basic use, it can be used for checkboxes and radiobuttons.
```
Comment (shahednasser) [inline documentation.html:364]: Please fix the spacing in words like "radiobuttons"

File: documentation.html [hunk 2]
```diff
@@ -352,6 +357,55 @@ <h4 class="display-inlineBlock">Icon Positioned to The Right</h4>
...
+          </div>
+          <br />
+          <section class="instruction-block" id="toggleCheckbox">
+            <h4 class="display-inlineBlock">Toggle Button as Checkboxes</h4>
+            <div class="text">
+              Adding the <code>toggle-btn toggle-off-btn</code> class to your checkbox inputs converts them to toggle buttons.For example:
```
Comment (shahednasser) [inline documentation.html:370]: `toggle-off-btn` is not required for checkboxes and radio buttons. Only for normal buttons

File: README.md [hunk 3]
```diff
@@ -26,6 +26,9 @@ Logo made by [Harshit Sharma](https://www.github.com/icoderharshit)
     - [Icon With No Background](#icon-with-no-background)
     - [Icon Positioned to The Left](#icon-positioned-to-the-left)
     - [Icon Positioned to The Right](#icon-positioned-to-the-right)
+  - [Toggle Button](#toggle-button)
+    -[Toggle Button as Checkboxes](#toggle-button-as-checkboxes)
```
Comment (shahednasser) [inline README.md:30]: There's a problem in the indentation please fix it (make sure it's similar to the previous menu items)

File: documentation.html [hunk 4]
```diff
@@ -352,6 +357,55 @@ <h4 class="display-inlineBlock">Icon Positioned to The Right</h4>
...
+              <br><br>
+              <h5>Which of the following is/are an OS?</h5><br>
+              <input type="checkbox" id="1" class="sbtn toggle-btn toggle-off-btn" />
```
Comment (shahednasser) [inline documentation.html:373]: The example should just be one checkbox with the code block for it. No need to include a full example (same thing for radio buttons) Also don't include a label as it is not necessary

File: documentation.html [hunk 5]
```diff
@@ -352,6 +357,55 @@ <h4 class="display-inlineBlock">Icon Positioned to The Right</h4>
...
+              <div class="script-copy">
+                <pre><code class="language-markup">&lt;input type="checkbox" class="sbtn toggle-btn toggle-off-btn" id="id" /&gt;&lt;label for="id"&gt;label&lt;/label&gt;</code></pre>
+                <div class="div-copy"><button class="clipboard"></button></div>
+              </div>
+            </div>
+          </section>
+          <section class="instruction-block" id="toggleRadio">
+            <h4 class="display-inlineBlock">Toggle Button as Radio Button</h4>
+            <div class="text">
+              Adding the <code>toggle-btn toggle-off-btn</code> class to your radio inputs converts them to toggle buttons.For example:
```
Comment (shahednasser) [inline documentation.html:390]: Add a space before "For example"

**Review by shahednasser — CHANGES_REQUESTED** [review]
Thank you for your work @VinayakChopra . I'd like to fix something about the documentation which is to indicate that the "toggle-off-btn" and "toggle-on-btn" are not necessary for checkboxes and radio buttons, they are just necessary for normal buttons. For checkboxes and radio buttons, it just depends on whether they're checked or not. Make sure to explain that in the documentation. 

I have also added some changes that should be done below. Please make sure that the changes indicated in the website should be made in the README as well.

**Comment by VinayakChopra** [comment]
Hi @shahednasser thanks for reviewing the work, I have made the said changes in the new commit.

**Comment by shahednasser** [comment]
Great thank you!

@all-contributors please add @VinayakChopra for documentation

**Pull request merged**